packages/expo-updates/.tests/e2e/__tests__/Updates-e2e-android.test.ts

```diff
@@ -0,0 +1,57 @@
+import spawnAsync from '@expo/spawn-async';
```

Comment:
Move this test file out of .tests -- we want these files to be visible. If they need to be excluded from the published npm package - a good idea - "files" in package.json or npmignore can do that.

Reply:
npmignore is generated by expo-module-scripts, but I've moved the `e2e` directory into the root package directory; it looks like all folders at any level named `__tests__` are excluded already so this will exclude everything but `e2e/jest.config.js`. LMK if you think it's worth adding "files" in package.json to exclude that too.

Reply:
I don't think it matters much then. If we add large files to the e2e folder that aren't excluded we can easily add an ignore rule.